cli/version/version_tools.go

```diff
@@ -16,6 +15,11 @@ const (
 	TypeBeta
 	TypeRC
 	TypeRelease
+
+	// VersionCliSeparator is used in commands to specify version. E.g: program=version.
```

Comment:
Oops. please remove "Version" prefix in comment for consts.

Reply:
changed